src/frontends/pytorch/src/op_table.cpp

```diff
@@ -299,6 +299,7 @@ OP_CONVERTER(translate_ones_fx);
 OP_CONVERTER(translate_ones_like_fx);
 OP_CONVERTER(translate_reflection_pad_nd_fx);
 OP_CONVERTER(translate_reshape_fx);
+OP_CONVERTER(translate_rrelu_fx);
```

Comment:
You also need to add the operation to the map: https://github.com/openvinotoolkit/openvino/blob/0d918dcebb02111194b8ec1dc43a055913a4141e/src/frontends/pytorch/src/op_table.cpp#L331